src/extractor/scripting_environment_lua.cpp

```diff
@@ -598,6 +615,9 @@ void Sol2ScriptingEnvironment::ProcessTurn(ExtractionTurn &turn)
         {
             turn_function(turn);
 
+            if( turn.weight == MAX_TURN_WEIGHT )
+              turn.weight = context.properties.GetMaxTurnWeight();
```

Comment:
you mean unconditionally cap the value? ie:
```c++
turn.weight = std::min(turn.weight, context.properties.GetMaxTurnWeight());`
```
what value should the profile use then to indicate max weight?